docs/source/jsonpath.rst

```diff
@@ -78,6 +78,42 @@ or::
 
 Examples like this can be found in one of gabbi's `own tests`_.
 
+When reading from disk you can apply the same JSONPath by adding a ':' to the
+end of your file name. This allows you to store multiple api responses into
```

Comment:
Small detail: For readability and consistency, please capitalize abbreviations like "API" and "JSON" (below).